pkg/lightning/config/config.go

```diff
@@ -613,6 +614,9 @@ func (cfg *Config) Adjust(ctx context.Context) error {
 			}
 			cfg.TikvImporter.DiskQuota = ByteSize(storageSize.Available - reservedSize)
 		}
+		if cfg.TikvImporter.DiskQuota < cfg.TikvImporter.LocalWriterMemCacheSize {
```

Comment:
should we solve `LocalWriterMemCacheSize` for `x = storageSize.Available - (enginesCount*x + writeAmount*autoDiskQuotaLocalReservedSpeed)`

and I'm not sure if this is to prevent any flush triggering disk quota. maybe we should also left a margin

Reply:
just removed this line of code 🙃